.github/workflows/helm-publish-new-package-version.yml

```diff
@@ -76,6 +76,10 @@ jobs:
       - name: Push tag
         run: git push origin $VERSION
 
+      - uses: actions/download-artifact@v3
+        with:
+          name: ${{ steps.package-helm-chart.outputs.package-file }}
+
```

Comment:
can't you expose the path as an output of helm-package-chart and use that instead of downloading again? you already have filename as an output

I know it's working as it is but `download-artifact` is really meant to be used to fetch an artifact produced in a previous job of a workflow - and ok that charts are small but you are consuming time and bandwidth for a thing that is already somewhere in the current workspace

Reply:
I agree it would be a better fix but I'm confused by the multiple usage of this action, and the README documents that downloading the artifact is the proper way to retrieve it.
I'd need to review the multiple callers in all repos, so I was tempted to revert the changes instead (eg the package is produced in current folder, as previously assumed). I'm not sure why this assumption was not present on https://github.com/Alfresco/alfresco-build-tools/blob/master/.github/actions/helm-release-and-publish/action.yml#L79

Reply:
@gionn I changed the logic, I will have to issue 4 more PRs to align on this (after release)